static/app/views/releases/detail/overview/commitAuthorBreakdown.tsx

```diff
@@ -85,27 +86,28 @@ class CommitAuthorBreakdown extends AsyncComponent<Props, State> {
 
     return (
       <Wrapper>
-        <SectionHeading>{t('Commit Author Breakdown')}</SectionHeading>
-        <Collapsible
-          expandButton={({onExpand, numberOfHiddenItems}) => (
-            <Button priority="link" onClick={onExpand}>
-              {tn(
-                'Show %s collapsed author',
-                'Show %s collapsed authors',
-                numberOfHiddenItems
-              )}
-            </Button>
-          )}
-        >
-          {sortedAuthorsByNumberOfCommits.map(({commitCount, author}, index) => (
-            <AuthorLine key={author?.email ?? index}>
-              <UserAvatar user={author} size={20} hasTooltip />
-              <AuthorName>{userDisplayName(author || {}, false)}</AuthorName>
-              <Commits>{tn('%s commit', '%s commits', commitCount)}</Commits>
-              <Percent>{this.getDisplayPercent(commitCount)}</Percent>
-            </AuthorLine>
-          ))}
-        </Collapsible>
+        <SidebarSection title={t('Commit Author Breakdown')}>
```

Comment:
Should the SidebarSection be moved from `app/components/group/` to `app/components/`?